### DISCUSSION TITLE
Timeout not taking effect, even on command

### DISCUSSION THREAD

**Discussion by mzerfas** [cypress-io/cypress#24038] [Component Testing]
`     cy.get('button', { timeout: 20000 })
          .contains('something')
          .should('be.visible')
          .click()`

^this is not working, still waits the default timeout for the overall config or test override, but individual timeouts don't work.  What am I missing?

**Comment by jjhelguero** [comment]
The timeout is applicable only to your `cy.get()`. You have a `.contains()` before the assertion so it might be best to condense the two into one [`.contains()`](https://docs.cypress.io/api/commands/contains)
```js
cy.contains('button', 'something', {timeout: 20000}).should('be.visible').click()
```
